pulp_2to3_migration/app/migration.py

```diff
@@ -61,6 +61,13 @@ def migrate_repositories(plan):
     )
     with ProgressReport(**progress_data) as pb:
         for plugin in plan.get_plugin_plans():
+            # all pulp2 repos in current plan were already migrated
+            if not Pulp2Repository.objects.filter(
+                is_migrated=False,
+                not_in_plan=False,
+                pulp2_repo_type=plugin.type
+            ):
```

Comment:
If we're not using the result, we should use ```.exists()```

Reply:
i will leave `filter`
```

In [7]: Pulp2Distributor.objects.filter(
   ...:             is_migrated=False,
   ...:             not_in_plan=False,
   ...: 
   ...:             )
Out[7]: <QuerySet [<Pulp2Distributor: pk=f26fac2e-1c65-4fd1-ac4e-8cff29a39aea>, <Pulp2Distributor: pk=e4eb10ee-5a4a-4ff5-b618-483a21d5b399>, <Pulp2Distributor: pk=fa4f38ff-790b-4738-9d5e-8a74517a080f>]>

In [8]: Pulp2Distributor.objects.exists(
   ...:             is_migrated=False,
   ...:             not_in_plan=False,
   ...: 
   ...:             )
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-7e9c7742da34> in <module>
      1 Pulp2Distributor.objects.exists(
      2             is_migrated=False,
----> 3             not_in_plan=False,
      4 
      5             )

/usr/local/lib/pulp/lib64/python3.6/site-packages/django/db/models/manager.py in manager_method(self, *args, **kwargs)
     80         def create_method(name, method):
     81             def manager_method(self, *args, **kwargs):
---> 82                 return getattr(self.get_queryset(), name)(*args, **kwargs)
     83             manager_method.__name__ = method.__name__
     84             manager_method.__doc__ = method.__doc__

TypeError: exists() got an unexpected keyword argument 'is_migrated'

In [9]: 
```